autode/transition_states/templates.py

```diff
@@ -452,3 +453,7 @@ def load(self, filename):
             raise TemplateLoadingFailed('Incorrect graph structure')
 
         return None
+
+    @property
+    def filename_str(self) -> str:
```

Comment:
shouldn't this be called `filename`?

Reply:
None-able attribute is already called filename 😕

Reply:
```suggestion
    def filename(self) -> str:
        return 'unknown' if self._filename is None else self._filename
```

Reply:
seems better to me to have the internal one have the dodgy name

Reply:
I hope it doesn't have any unintended side effects...

Reply:
just looks – seems okay(?)